test/distributed/test_c10d_common.py

```diff
@@ -247,12 +247,20 @@ def _multi_worker_helper(self, world_size):
     @unittest.skipIf(
         IS_WINDOWS, "Skip test for windows due to multiprocessing library error when using windows spawn"
     )
+    @unittest.skipIf(
```

Comment:
Instead of disabling these tests, can't we just use `spawn` in _multi_worker_helper?